### DISCUSSION TITLE
How can I get learning rate for each epoch?

### DISCUSSION THREAD

**Discussion by OldWheat** [Lightning-AI/pytorch-lightning#15868] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I have defined a learning rate scheduler. I want to get the learning rate for each epoch and record it on the tenosrboard. How do I do this? Thanks
